http-client/src/test/java/io/airlift/http/client/TestHttpClientConfig.java

```diff
@@ -239,4 +246,16 @@ private List<String> getJettyDefaultExcludedCiphers()
         SslContextFactory sslContextFactory = new SslContextFactory.Client();
         return Arrays.asList(sslContextFactory.getExcludeCipherSuites());
     }
+
+    @VisibleForTesting
+    static DataSize clamp(DataSize value, DataSize min, DataSize max)
```

Comment:
drop this one if you already expose version from `HttpClientConfig`

Reply:
Dropped